include/circt/Dialect/HW/HWStructure.td

```diff
@@ -16,8 +16,7 @@ class HWModuleOpBase<string mnemonic, list<Trait> traits = []> :
     HWOp<mnemonic, traits # [
       DeclareOpInterfaceMethods<HWModuleLike>,
       DeclareOpInterfaceMethods<HWMutableModuleLike>,
-      FunctionOpInterface, Symbol,
-      OpAsmOpInterface, HasParent<"mlir::ModuleOp">]> {
+      FunctionOpInterface, Symbol, OpAsmOpInterface]> {
```

Comment:
What if we don't include the external HW operation directly in the calyx.program? Does that have different repercussions? At least in the short term, that would have the benefit of not affecting the HW dialect.

Reply:
I'm not sure how this would affect the CalyxEmitter. Currently the CalyxEmitter only considers operations within a calyx.program from what I understand.

Reply:
I think the CalyxEmitter can just as easily consider for builtin::ModuleOp as it can look for calyx::ProgramOp.